Assets/Settings/AetherInputSystem.inputactions

```diff
@@ -44,6 +44,22 @@
                     "expectedControlType": "",
                     "processors": "",
                     "interactions": "Press"
+                },
+                {
+                    "name": "LaunchMeteor",
+                    "type": "Button",
+                    "id": "c6ceacf3-e928-4f63-af94-a834b11e20f5",
+                    "expectedControlType": "",
+                    "processors": "",
+                    "interactions": "Press"
+                },
+                {
+                    "name": "LaunchForcefield",
+                    "type": "Button",
+                    "id": "3ab4c3fe-e4d1-4d22-b2ce-3e03d660aad3",
+                    "expectedControlType": "",
+                    "processors": "",
+                    "interactions": "Press"
```

Comment:
Let's not create specialized inputs in the input system. Instead, we should create generic skill trigger buttons and check against those instead. Otherwise, every skill will start adding to the input system.

If this change is for debug, keep it in your branch and don't commit it